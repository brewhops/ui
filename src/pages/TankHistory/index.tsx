import React from 'react';

export const TankHistory: React.FC = () => {
  return <div>Tank History</div>
}
  