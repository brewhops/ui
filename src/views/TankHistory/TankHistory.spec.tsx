import React from 'react';
import { TankHistory } from './TankHistory';
import { shallow } from 'enzyme';

describe('src/pages/TankHistory/TankHistory.tsx', () => {
  it('renders the Tank History page', () => {
    expect(shallow(<TankHistory/>)).toMatchSnapshot();
  })
});