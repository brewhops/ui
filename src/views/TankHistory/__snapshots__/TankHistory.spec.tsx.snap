// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src/pages/TankHistory/TankHistory.tsx renders the Tank History page 1`] = `ShallowWrapper {}`;
